src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,14 +35,16 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        User user = userService.findByEmail(authentication.getName()).get();
```

Comment:
I guess you saw a warning about using method get() so better throw an exception.